test/tests/ver-1hb/ver-1hb.i

```diff
@@ -0,0 +1,140 @@
+# R = '${units 8.31446261815324 J/K/mol}' # from PhysicalConstants
+# T = '${units 303 K}'
+# N_a = '${units 6.02214076e23 at/mol}' # from PhysicalConstants
```

Comment:
Let's remove those if they are not needed.

Reply:
Or maybe add the concentration calculations that use them. That's probably the better option.

Reply:
Since we don't plot the concentration variables I think it is better to remove them. We already have an example that shows how to do the conversion where we actually compare concentrations.